src/components/WebexSettings/Select.scss

```diff
@@ -0,0 +1,10 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-select {
+  select {
+    font-family: "CiscoSansTT Regular","Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
isn't the font family already set globally?

Reply:
Yes, it is. I'll remove this line.